pkg/proxy/backup.go

```diff
@@ -251,11 +253,69 @@ func (p *Proxy) BackupVolumeList(ctx context.Context, req *rpc.EngineBackupVolum
 		Volumes: map[string]*rpc.EngineBackupVolumeInfo{},
 	}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.